src/snowflake/snowpark/_internal/analyzer/snowflake_plan.py

```diff
@@ -266,6 +267,14 @@ def __init__(
         # UUID for the plan to uniquely identify the SnowflakePlan object. We also use this
         # to UUID track queries that are generated from the same plan.
         self._uuid = str(uuid.uuid4())
+        self._attributes = None
```

Comment:
add type hint and comment for those two fields

Reply:
fixed